docs/CucumberSyntax.md

```diff
@@ -22,11 +22,11 @@ More examples can be seen in the [generator cucumber features](https://github.co
 The framework supports setting configuration settings for the generator, defining the profile and describing the expected outcome. All of these are described below, all variable elements (e.g. `{generationStrategy}` are case insensitive), all fields and values **are case sensitive**.
 
 ### Configuration options
-* _the generation strategy is `{generationStrategy}`_ see [generation strategies](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `full`
-* _the combination strategy is `{combinationStrategy}`_ see [combination strategies](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `pinning`
-* _the walker type is `{walkerType}`_ see [walker types](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `cartesian_product`
+* _the generation strategy is `{generationStrategy}`_ see [generation strategies](https://github.com/ScottLogic/datahelix/blob/master/generator/docs/GenerationTypes.md) - default: `random`
+* _the combination strategy is `{combinationStrategy}`_ see [combination strategies](https://github.com/ScottLogic/datahelix/blob/master/generator/docs/CombinationStrategies.md) - default: `pinning`
+* _the walker type is `{walkerType}`_ see [walker types](https://github.com/ScottLogic/datahelix/blob/master/generator/docs/TreeWalkerTypes.md) - default: `cartesian_product`
 * _the data requested is `{generationMode}`_, either `violating` or `validating` - default: `validating`
-* _the generator can generate at most `{int}` rows_, ensures that the generator will only emit `int` rows, default: `10000000` (10 million)
+* _the generator can generate at most `{int}` rows_, ensures that the generator will only emit `int` rows, default: `1000`
```

Comment:
Default is `200` actually, but only in cucumber, the generator will default to `1000` but we don't need this much data to assert the correct behaviour in cucumber. This should have been updated in #878